### PR TITLE
Better Contento connection info in error

### PR DESCRIPTION
**Pull request by paulanthonywilson**
When trying to run Front End from development, there's a few things that
you need to have set up to get a connection to Contento. This is written
up in the README, but if someone's missed something it's not that easy
to tell them what they've missed.

This is an attempt to add a bit more context for debugging this error.

### CODE REVIEW

**Comment by spencer** [comment]
:+1: Useful.

File: lib/core/repository/categories/cms.rb [hunk 1]
```diff
@@ -6,10 +6,15 @@ def initialize(options = {})
...
+        response = connection.get(categories_url)
         response.body
-      rescue
-        raise RequestError, 'Unable to fetch Category JSON from Contento'
+      rescue => e
+        raise RequestError,
+              'Unable to fetch Category JSON from Contento' +
+              " url: [#{categories_url}]" +
+              " error: [#{e.inspect}]" +
+              " url_prefix: [#{connection.try(:url_prefix).inspect}]"
```
Comment (asmega) [inline lib/core/repository/categories/cms.rb]: where does `url_prefix` come from?

Comment (paulanthonywilson) [inline lib/core/repository/categories/cms.rb]: The connection, or ultimately `ENV['MAS_CMS_URL']. I was logging out the whole connection but it was a bit noisy.

This is what you get with this (in dev mode, obviously).

voila_capture 2015-11-11_10-50-40_am

Comment (asmega) [inline lib/core/repository/categories/cms.rb]: figured it out, it's from faraday.

**Comment by asmega** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:

**Comment by paulanthonywilson** [comment]
Thanks @spencer, @asmega, @tomas-stefano.

**Pull request merged**